src/SFA.DAS.ProviderRequestApprenticeTraining.Infrastructure/Api/IProviderRequestApprenticeTrainingOuterApi.cs

```diff
@@ -7,6 +8,9 @@ namespace SFA.DAS.ProviderRequestApprenticeTraining.Domain.Interfaces
 {
     public interface IProviderRequestApprenticeTrainingOuterApi
     {
+        [Get("/employerrequests/provider/{ukprn}/email-addresses/{userEmailAddress}")]
```

Comment:
This isn't (I think) a rest type of Url as the userEmailAddress is not one from a collection of email-address like the ukprn is one from a collection of providers - the userEmailAddress is a query parameter

Reply:
Marked userEmailAddress as a query parameter